TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Shooter.java

```diff
@@ -0,0 +1,26 @@
+package org.firstinspires.ftc.teamcode;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.Gamepad;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+
+public class Shooter {
+
+    String DRIVE_TRAIN_CAPTION = "Drive Status";
+    Telemetry telemetry;
+    HardwareInnov8Robot robot;
```

Comment:
you can make a constructor for this class that initializes `telemetry`, `robot`, and `opmode`

Reply:
I don't know if I did this right but I made a constructor.